### ISSUE TITLE
Changes in this gantt chart package

### ISSUE THREAD

**Issue by developerfix** [Bdaya-Dev/flutter_gantt_chart#3]
Hi guys,
My client asked for a flutter gantt chart and send me the following image as an example. So considering this https://pub.dev/packages/gantt_chart the only package available for gantt chartt right now. Is it possible to make changes in this package and make it like the one in the image or similar to it. I mean those little arrows and view of the screens should be of 1 month, 3 months ,6 and 12 months only and user can click on the right left arrows to see more instead of horizontal scrolling.

kindly please answer.
@mauriziopinotti 
@ahmednfwela 

Thanks

here is the desired output:
![image](https://user-images.githubusercontent.com/60823479/178146493-b7d5e390-4510-4af6-8e36-9aa2a4ace5ed.png)

**Comment by ahmednfwela** [comment]
for the left, right arrows I have to change gantt view to take a ScrollController as an optional input.
as for the customizable month range, it's doable (it's already being done for 12 months).

however I am not sure about the vertical lines (checkpoints?) and the sub tasks

**Comment by developerfix** [comment]
for the left, right arrows I have to change gantt view to take a ScrollController as an optional input.
as for the customizable month range, it's doable (it's already being done for 12 months).


I mean i dont want to be scrollable list. i want to be fixed 1month,3 months,6 months or 12 months of table showing on the screen. In the bottom right corner of the above image, you can see the slider for the month select for view. Now for example if user selects 3 month view and timeline for the tasks go beyond 3 months, user will click on the right arrow and next 3 months will show up and so on. I hope you got my point.

In short, I want arrows to be workable like that.
another thing you said 12 month view is already being done? can you please tell where can i see it?

Thanks
